docs/tutorials/experimental-fits.md

```diff
@@ -0,0 +1,436 @@
+    La_up   = np.sqrt(kappa_a * thermal_a) * dq.dag(a)
+    Lb_down = np.sqrt(kappa_b * (1 + thermal_b)) * b
+    Lb_up   = np.sqrt(kappa_b * thermal_b) * dq.dag(b)
+
+    result = dq.mesolve(
+        H, [Lb_down, Lb_up, La_down, La_up],
+        rho0, time,
+        exp_ops=[dq.dag(a) @ a],
+    )
+
+    return result
+```
+
+We choose an initial guess that is reasonable with respect to the data we have. We converge to a steady state in roughly $1.0\mu$s so $g_2$ should be of the order of $1$MHz. Let's take $0.1$MHz to have some margin.
+```python
+g2 = jnp.array(0.1 * MHz)
+result = simulate(g2)
+
+plt.figure(figsize=(10, 5))
+
+time_np = time
+for i in range(len(alpha2)):
+    plt.plot(time_np, data[:, i].real, f"+C{i}")
+    plt.plot(time_np, result.expects[i, 0].real, f"C{i}")
+plt.grid()
+plt.xlabel("Time [us]")
+plt.ylabel("Photon number")
+renderfig('g2-initial-guess')
+```
+![g2 initial guess](/figs-docs/g2-initial-guess.png)
+
+
+% skip: start
+
+The optimization loop is defined as follow
+```python
+def cost(g2, data):
+    res = simulate(g2)
+    return jnp.sum((data - res.expects[:, 0].real.T) ** 2)
+
+g2 = jnp.array(0.1 * MHz)
+costs = []
+g2s = [g2]
+for _ in tqdm(range(40)):
+    val, grad = jax.value_and_grad(cost)(g2, data)
+    grad = jnp.clip(grad, -100, 100)
+    g2 = g2 - 0.01 * grad
+    costs.append(val)
+    g2s.append(g2)
+costs.append(cost(g2, data))
+
+costs = np.array(costs)
+g2s = np.array(g2s)
+```
+we use a plain gradient descent algorithm for simplicity but one could imagine using any other algorithm, such as the ones provided in [Optax](https://github.com/google-deepmind/optax).
+
+!!! note Optimisation time
+    The full optimisation runs in less than one minute on a NVIDIA GeForce RTX 4090 and around 5 minutes on a M1 Mac.
+
+
+We can plot the loss to ensure we converged correctly
+```python
+plt.figure(figsize=(10, 5))
+plt.subplot(121)
+plt.plot(costs)
+plt.grid()
+plt.xlabel("iteration")
+plt.ylabel("loss")
+plt.yscale("log")
+
+plt.subplot(122)
+plt.plot(g2s / MHz)
+plt.grid()
+plt.xlabel("iteration")
+plt.xlabel("g2 [MHz]")
+```
+![Fit result](/figs-docs/g2-fit-result.png)
+
+Finally we plot the fit and compare with the data
+```python
+result = simulate(g2)
+plt.figure(figsize=(10, 5))
+
+time_np = time
+for i in range(len(alpha2)):
+    plt.plot(time_np, data[:, i].real, f"+C{i}")
+    plt.plot(time_np, result.expects[i, 0].real, f"C{i}")
+plt.grid()
+plt.xlabel("Time [us]")
+plt.ylabel("Photon number")
+
+print(g2 / MHz)
+```
+
+!!! example "Final note"
+    $g_2$ value differs from the one reported in
+    [arxiv:2307.06617](https://arxiv.org/pdf/2307.06617.pdf)
+    by approximately 15%. This discrepancy is attributed to the unavailability 
+    of Dynamiqs for performing fits at the time of writing the article. 
+    Consequently, the authors were unable to employ gradient descent for their
+    fit and had to resort to alternative methods.
+
+
+## Full code
```

Comment:
Ideally we'll have a link at some point for this, but for now it's fine I think.